rust/agama-lib/src/proxies/locale.rs

```diff
@@ -20,7 +20,8 @@
 //! [D-Bus standard interfaces]: https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces,
 use zbus::proxy;
 #[proxy(
-    default_service = "org.opensuse.Agama1",
+    default_service = "org.opensuse.Agama.Manager1",
+    default_path = "/org/opensuse/Agama/Manager1",
```

Comment:
Good catch... except it does not work: :rofl: :cry: 

When I run this code and change the UI langue in the browser, `journalctl` has:

> Nov 12 13:10:23 c75fbede8655 agama-web-server[28278]: Sending message: Msg { type: MethodCall, serial: 157, sender: UniqueName(":1.7"), path: ObjectPath("/org/freedesktop/Locale"), iface: InterfaceName("org.freedesktop.DBus.Properties"), member: MemberName("Set"), body: Structure(Dynamic { fields: [Str, Str, Variant] }), fds: [] }
...
Nov 12 13:10:23 c75fbede8655 agama-web-server[28278]: 5:185m5:185mCould not synchronize settings in the localization D-Bus service: D-Bus service error: org.freedesktop.DBus.Error.UnknownObject: Unknown object '/org/freedesktop/Locale'

Looking up "Could not synchronize settings" in the source, I arrive at a `update_dbus`  fn, which uses a `set_locales`  fn, defined in... https://github.com/agama-project/agama/blob/65939ce64a7bb117ed66f37ebae814b61376f871/rust/agama-lib/src/localization/proxies.rs#L62 

Which is a similar, yet different file from this diff
:shrug:

Reply:
Well, it works for me (TM) 😉 Actually, I would not expect the `set_locales` method to be involved because that's responsible for changing the locales to install, not the installer's language. Are you changing the language through the "Installer Options" menu? Did you restart the `agama-dbus-server` process too (restarting the `agama.service` should be enough)?

Having two interfaces with the same name (`org.opensuse.Agama1.Locale`) is unfortunate, I know, but they are different things:

* [localization/proxies](https://github.com/agama-project/agama/blob/65939ce64a7bb117ed66f37ebae814b61376f871/rust/agama-lib/src/localization/proxies.rs#L62) refers to the Locale API (implemented in `agama-dbus-server`). We kept a D-Bus interface because it was needed for internal communication. But the idea is to get rid of it and rely only on HTTP.
* [proxies/locale](https://github.com/agama-project/agama/blob/65939ce64a7bb117ed66f37ebae814b61376f871/rust/agama-lib/src/proxies/locale.rs) refers to an interface that is implemented by all Ruby-based services which allows to change the language.

I wonder if we should add `default_path = "/org/opensuse/Agama1/Locale",` to the 1st proxy, but in my testing system it seems to work (unless I am missing something).

In any case, I will build an image that includes all the changes so we can perform more tests.

Reply:
I have fixed both proxies and now I see no error in the logs.

Reply:
> Having two interfaces with the same name (org.opensuse.Agama1.Locale) is unfortunate, I know, but they are different things:

It's not unfortunate, it is **wrong**  :smile: but it happens to work now. I hope I can add a simple fix, otherwise postpone it for another PR

Reply:
Yes, it is wrong 😅 The point is that we plan to drop one of them, so it won't be duplicated anymore.